### PR TITLE
Add support for SimpleBlobDetectorParams

### PR DESCRIPTION
**Pull request by ryonsabouni**
Needed this functionality, and have started to use it. Open to any and all suggestions!

### CODE REVIEW

File: features2d.go [hunk 1]
```diff
@@ -350,13 +355,213 @@ func NewSimpleBlobDetector() SimpleBlobDetector {
 	return SimpleBlobDetector{p: unsafe.Pointer(C.SimpleBlobDetector_Create())}
 }
 
+// NewSimpleBlobDetectorWithParams returns a new SimpleBlobDetector with custom parameters
```
Comment (deadprogram) [inline features2d.go]: Can you please add the link to the OpenCV docs, same as that used by `NewSimpleBlobDetector()`? Thank you.

**Comment by deadprogram** [comment]
This is really great contribution @ryonsabouni thank you for working on it. I made one small comment regarding the GoDocs. Otherwise looks good to merge.

File: features2d.go (hunk 1 above)
Comment (ryonsabouni) [reply features2d.go]: Added

**Comment by deadprogram** [comment]
Thank you very much for the PR and the update. Now merging.

**Pull request merged**